packages/wallet/src/__stories__/index.tsx

```diff
@@ -13,6 +15,31 @@ const walletStateRender = state => () => {
   );
 };
 
+export const protocolStateRender = (protocolState: ProtocolState) => {
```

Comment:
I am in favour of storybook using the top level `Wallet` container to render `protocolState`s (as it does here), so that we get a true representation of what will be shown. I guess we need to embed the protocolState more judiciously than is currently done here, so that the `Wallet` container can infer what to display (as it would in the wild). 

Perhaps this function should become
```typescript
export const topLevelProtocolStateRender = (protocolState: TopLevelProtocolState) => {
```
 and have some cascading render functions that embed a child state on an appropriate parent state before calling into ` topLevelProtocolStateRender`. For example: 

```typescript
export const renderAsChildOfConcluding = (defundingState: DefundingState) => {
    const protocolState = concluding.waitForDefund({...defaults, defundingState}),
    return topLevelProtocolStateRender(protocolState);
}
```